dynawo/sources/Modeler/ModelManager/DYNModelManager.cpp

```diff
@@ -487,6 +492,7 @@ ModelManager::initParams() {
   solveParameters();
   try {
     checkDataCoherence(getCurrentTime());
+    checkParametersCoherence();
   } catch (const MessageError& Msg) {
```

Comment:
I'm not very sure that we need these two additional checks here in addition to the ones called by the DYNSimulation file. What is your opinion?